routes/api/v1/meals.js

```diff
@@ -0,0 +1,28 @@
+var express = require('express');
+var router = express.Router();
+var Meal = require('../../../models').Meal;
+var Food = require('../../../models').Food;
+var MealFood = require('../../../models').MealFood;
+var defaultHeader = ["Content-Type", "application/json"];
+
+router.post('/:meal_id/foods/:id', async function(req, res, next){
+  console.log("HERE", req.params);
+  try{
+    let meal = await Meal.findByPk(req.params.meal_id);
+    let food = await Food.findByPk(req.params.id);
+    if (meal != null && food != null){
+      await MealFood.create({FoodId: food.id, MealId: meal.id})
+      res.setHeader(...defaultHeader);
+      let message = `Successfully added ${food.name} to ${meal.name}`
+      console.log(message);
```

Comment:
I'm still unclear as to whether or not we should leave console.log(s) in or take them out like debugger break points but just in case there's one here.